docs/index.md

```diff
@@ -1,17 +1,78 @@
-* `mkdocs serve` - Start the live-reloading docs server.
-* `mkdocs build` - Build the documentation site.
-* `mkdocs -h` - Print help message and exit.
+Melange is a fork of the OCaml compiler that compiles to JavaScript. Melange
+strives to provide the best integration with both the OCaml and JavaScript
+ecosystems. To know more about it start by reading the [introductory
+tutorial](getting-started.md), then check the [Learn](learn.md) section for more
+information.
 
-## Project layout
+<div class="text-center">
+<a href="getting-started" class="btn btn-primary" role="button">Getting Started</a>
+<a href="learn" class="btn btn-primary" role="button">Learn</a>
+</div>
 
-    mkdocs.yml    # The configuration file.
-    docs/
-        index.md  # The documentation homepage.
-        ...       # Other markdown pages, images and other files.
+<div class="jumbotron">
+<h2 class="display-4 text-center">Features</h2>
+
+<div class="row">
+  <div class="col-sm-6">
+    <div class="card">
+      <div class="card-body">
+        <h3 class="card-title">A Solid Type System</h3>
+        <p class="card-text">
+            Melange leverages OCaml's powerful type system to catch more bugs at
+            compile time. Large, complex codebases become easy to maintain and
+            refactor.
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="col-sm-6">
+    <div class="card">
+      <div class="card-body">
+        <h3 class="card-title">First-Class Editor and Tooling</h3>
+        <p class="card-text">
+            Melange can use the same editor integrations as OCaml, which exist
+            for VSCode, Vim, or Emacs. They allow to provide type inspection,
+            autocomplete and more. It also has first-class integration with <a
+            href="https://dune.build/">Dune</a>, OCaml's most used build system.
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="row">
+  <div class="col-sm-6">
+    <div class="card">
+      <div class="card-body">
+        <h3 class="card-title">JavaScript Integration</h3>
+        <p class="card-text">
+            Whether you want to use existing JavaScript packages from NPM, or
+            use your own JavaScript libraries in your projects, Melange has you
+            covered. With an expressive bindings language, and an ergonomic
+            compilation model, Melange can help you build robust applications
+            that leverage functionality from the JavaScript ecosystem.
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="col-sm-6">
+    <div class="card">
+      <div class="card-body">
+        <h3 class="card-title">Stable and Industry Backed</h3>
+        <p class="card-text">
+            Melange builds on top of decades of type system research, compiler
+            engineering and tooling development to provide a polished
+            developer experience. Companies like Ahrefs use Melange daily to
+            deploy web applications to their users.
```

Comment:
for their users

Reply:
https://github.com/melange-re/documentation-site/commit/99497aac309ed78d28cee692357d6b994304f770